new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWITValidation.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes;
+
+import oracle.weblogic.kubernetes.actions.TestActions;
+import oracle.weblogic.kubernetes.actions.impl.primitive.WITParams;
+import oracle.weblogic.kubernetes.actions.impl.primitive.WebLogicImageTool;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@DisplayName("Simple validation of basic WIT functions")
+class ItWITValidation implements LoggedTest {
+  @Test
+  @DisplayName("Create a MII image")
+  public void testCreatingMIIImage() {
+
+    // install WIT using the default version and location
+    boolean downloadWIT = TestActions.installWIT(
+        null /* version */, 
+        null /* location */,
+        false /* do not redirect output */);
+   
+    assertEquals(true, downloadWIT);
```

Comment:
include messages on asserts, so if they fail we get a descriptive message

Reply:
will do

Reply:
done